### PR TITLE
PARQUET-582: Conversions functions for Parquet enums to Thrift enums

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by xhochy** [comment]
I'm not totally done yet with write support but to make reviewing it easier, I'm trying to split it a bit up into independent parts that can be already reviewed & merged.

File: src/parquet/thrift/util.h [hunk 1]
```diff
@@ -48,6 +48,27 @@ static inline Compression::type FromThrift(format::CompressionCodec::type type)
   return static_cast<Compression::type>(type);
 }
 
+static inline format::Type::type ToThrift(Type::type type) {
+  return static_cast<format::Type::type>(type);
+}
+
+static inline format::ConvertedType::type ToThrift(LogicalType::type type) {
+  // item 0 is NONE
```
Comment (wesm) [inline src/parquet/thrift/util.h]: Add a `DCHECK_NE(type, LogicalType::NONE)`

Comment (xhochy) [inline src/parquet/thrift/util.h]: Done.

**Comment by wesm** [comment]
+1, thank you

**Pull request closed without merging**